Model/ResourceModel/Magazine.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * @package Styla/Connect2
+ * @author Oskar Wolanin <owolanin@divante.co>
+ * @copyright 2018 Divante Sp. z o.o.
+ * @license See LICENSE_DIVANTE.txt for license details.
+ */
+
+namespace Styla\Connect2\Model\ResourceModel;
+
+
+class Magazine extends \Magento\Framework\Model\ResourceModel\Db\AbstractDb
+{
+    /**
+     * @return void
+     */
+    protected function _construct()
+    {
+        $this->_init('styla_magazine', 'id');
+    }
+
+    /**
+     * @param string $field
+     * @param int|string $value
+     * @param \Magento\Framework\Model\AbstractModel $object
+     *
+     * @return \Magento\Framework\DB\Select
+     * @throws \Magento\Framework\Exception\LocalizedException
+     */
+    protected function _getLoadSelect($field, $value, $object)
```

Comment:
To chyba jest wykonywane przy $model->load(); Osobiście zrobiłbym w tym resource osobną metodę getXXX i tam zawarł logikę, a nastepnie w klasie która ma zależność do resouceModelu odwoływałbym się do niej przez $this->magazineResouceModel->getXxx()

Reply:
Sprawdzę to jak przeniosę całą funkcjonalność na M2.